### DISCUSSION TITLE
A luz azul não está piscando.    The blue light is not flashing.

### DISCUSSION THREAD

**Discussion by Alefox007** [darthcloud/BlueRetro#537] [Q&A] [answered]
Olá, estou com problemas no meu BlueRetro e a culpa é minha, meu aparelho está na versão 1.7 e fui atualizar para a 1.8 porém cometi algum erro e agora a luz azul de pareamento está travada, não está piscando, e o meu BlueRetro está e inutilizável, algum pode me ajudar a como consertar isso?

![IMG_20230111_231457](https://user-images.githubusercontent.com/122080657/211959909-3d623e3d-1867-40b3-a3b3-ce5df9184227.jpg)

**Comment by Alefox007** [comment]
**I'm Brazilian and I'm not good at the English language, so I'm leaving this comment with this Google translator translation below:

Hello, I'm having problems with my BlueRetro and it's my fault, my device is on version 1.7 and I've been updated to 1.8 but I made some mistake and now the blue pairing light is stuck, it's not blinking, and my BlueRetro is unusable , can anyone help me how to clarify this**

**Comment by darthcloud** [comment, marked as answer]
1. Download the v1.8.1_hw1.zip from https://darthcloud.itch.io/blueretro
2. Extract the zip file
3. The 3rd file to use is: BlueRetro_universal.bin
4. Follow those instructions: https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

**Reply by Alefox007** [reply]
Olá, muito obrigado, eu consegui atualizar para versão 1.8.0. A versão 1.8.1 não funcionou, a luz azul pisca 2x e depois apaga.

**Hello, thank you very much, I managed to update to version 1.8.0. Version 1.8.1 didn't work, the blue light flashes 2x and then goes off.**
